control-plane/agents/src/bin/core/controller/scheduling/resources/mod.rs

```diff
@@ -145,6 +152,10 @@ impl ReplicaItem {
     pub(crate) fn child_info(&self) -> Option<&ChildInfo> {
         self.child_info.as_ref()
     }
+    /// Count of vg replicas on the replica's pool.
+    pub(crate) fn vg_replica_count_on_pool(&self) -> u64 {
```

Comment:
```suggestion
    pub(crate) fn vg_replicas_on_pool(&self) -> u64 {
```
match struct name for consistency